bloom/generators/debian/generator.py

```diff
@@ -417,6 +417,20 @@ def generate_substitutions_from_package(
     data['debhelper_version'] = 7 if os_version in ['oneiric'] else 9
     # Summarize dependencies
     summarize_dependency_mapping(data, depends, build_depends, resolved_deps)
+    # Copyright
+    licenses = []
+    separator = '\n' + '=' * 80 + '\n\n'
```

Comment:
Why this choice for the license separator?

Reply:
Because I just found some packages use this as a separator.
But now I found the "\n----*\n\n" is used more than "\n===*\n\n" in my computer. Hmm

```bash
/usr/share/doc⟫ grep -PHzol "\n\n======*\n\n" */copyright | wc -l
24
/usr/share/doc⟫ grep -PHzol "\n\n------*\n\n" */copyright | wc -l
55
```

Do you have better idea?

Reply:
Given that there is no overriding convention, I have no preference. I think either convention is fine and it can be left as-is.